### DISCUSSION TITLE
Explain in the documentation what time units are valid for mutes

### DISCUSSION THREAD

**Discussion by zambam5** [jagrosh/Vortex#87] [Questions] [answered]
There is no clear explanation on what units or timeframes are allowed

`>>mute @[username] 7d` will return `[username] muted for 1w` but `>>mute @[username] 1w` does not work.

**Comment by jagrosh** [comment, marked as answer]
Valid units of time are **seconds**, **minutes**, **hours**, and **days**. These can be combined in any order, and the input validation for these is quite flexible; all of the following are valid input for time:
```
2hr30min
4 minutes and 10 seconds
2d 1 minute
6sec and 2 days
```
I will update the wiki with this in some manner (either in the full command reference or as a referenced page) soon
